packages/zapp/console/src/components/Entities/generators.ts

```diff
@@ -40,3 +40,18 @@ export const backUrlGenerator: {
   [ResourceType.UNSPECIFIED]: workflowListGenerator,
   [ResourceType.WORKFLOW]: workflowListGenerator,
 };
+
+const workflowDetailGenerator = ({ project, domain, name }: ResourceIdentifier) =>
+  Routes.WorkflowDetails.makeUrl(project, domain, name);
+const taskDetailGenerator = ({ project, domain, name }: ResourceIdentifier) =>
+  Routes.TaskDetails.makeUrl(project, domain, name);
+
+export const backToDetailUrlGenerator: {
+  [k in ResourceType]: (id: ResourceIdentifier) => string;
+} = {
+  [ResourceType.DATASET]: workflowDetailGenerator,
+  [ResourceType.LAUNCH_PLAN]: workflowDetailGenerator,
+  [ResourceType.TASK]: taskDetailGenerator,
+  [ResourceType.UNSPECIFIED]: workflowDetailGenerator,
+  [ResourceType.WORKFLOW]: workflowDetailGenerator,
```

Comment:
is using workflowDetailGenerator for all types correct here, or would it be better to either make it optional or create a dummy one that throws a not implemented error?

Reply:
throw the error if not implmented! tks